native/spark-expr/src/bloom_filter/spark_bit_array.rs

```diff
@@ -102,8 +101,10 @@ impl SparkBitArray {
     }
 }
 
+/// Returns the number of 64-bit words needed to store a `num_bits`-bits value
+#[inline]
 pub fn num_words(num_bits: i32) -> i32 {
-    bit::ceil(num_bits as usize, 64) as i32
+    ((num_bits as usize) / 64 + ((num_bits % 64 != 0) as usize)) as i32
```

Comment:
What is the purpose of this change? It seems to be manually inlining the `bit::ceil` function?

```rust
pub fn ceil(value: usize, divisor: usize) -> usize {
    value / divisor + ((value % divisor != 0) as usize)
}
```

Reply:
No important reason. It is a duplicate from the core crate which is not a dependency. The only use case of the ceil here is for `num_words` so I thought it should belong directly there.

Will extract this